src/networks.ts

```diff
@@ -418,4 +418,26 @@ export const networkInfo = new Map<number, NetworkInfo>([
       stagingBaseAPI: 'https://transaction-testnet.staging.safe.kava.io',
     },
   ],
+  [
+    660279,
+    {
+      chainID: 660279,
+      name: 'Xai',
+      shortName: 'xai',
+      currencySymbol: 'XAI',
+      baseAPI: 'https://transaction.safe.xai.games',
+      stagingBaseAPI: 'https://transaction-testnet.safe.xai.games',
```

Comment:
```suggestion
      stagingBaseAPI: 'https://transaction.staging.safe.xai.games',
```
This one should for Mainnet staging